supplementary_style_guide/style_guidelines/formatting.adoc

```diff
@@ -8,6 +8,21 @@ Follow the _IBM Style_ guide advice of _day Month year_ for date formats, for ex
 
 When the format _day Month year_ causes a presentation or clarity issue, use _Month day, year_ (for example, October 3, 2019) instead.
 
+[[man-pages]]
+== Man page references
+
+When referencing a man page in an additional resources section, use the following format: `_<name_of_utility>(<section_number>)_` _man page on your system_.
```

Comment:
1. Should we highlight "Additional resources" in some way?
2. Would the text be easier to read if the format was on a new line?

Reply:
1. Technical nitpick: `<name_of_utility>` is only correct for section 8 and 1. There are also man pages for config files `NetworkManager.conf(5)`, system calls `exit(2)`, libraries `exit(3)`, and others . I would recommend `<man_page_name>(<section_number>)`
2. Is there a special reason for italic on "man page on your system"? The [ISG](https://www.ibm.com/docs/en/ibm-style?topic=references-citations#general-guidelines) does not use italic for qualifiers (`For more information, see the _DQTI_ book.`)

Reply:
@libander I don't understand your comment. How is the `man` man page content you posted relevant for the proposed change in the SSG?

Reply:
> 1. Technical nitpick: `<name_of_utility>` is only correct for section 8 and 1. There are also man pages for config files `NetworkManager.conf(5)`, system calls `exit(2)`, libraries `exit(3)`, and others . I would recommend `<man_page_name>(<section_number>)`

Likewise, given that the man pages themselves classify the pages: 
> Each page argument given to man is normally the name of a program, utility or function.

I'd consider changing line 16's: 
`to view the information for the named command or utility.` 
to: 
`to view the information for the program, utility or function.`